src/elasticsearch/user-elasticsearch.service.ts

```
@@ -95,6 +95,7 @@ export class UserElasticsearchService implements OnModuleInit {
                   type: 'keyword',
                   normalizer: 'country_keyword_normalizer',
                 },
+                auto_tags: { type: 'keyword' },
```

ESLint

[error] Replace `'keyword'` with `"keyword"`

(prettier/prettier)